### ISSUE TITLE
OverflowError in jointplot()

### ISSUE THREAD

**Issue by conduit242** [mwaskom/seaborn#499]
I have two equal length integer arrays which are causing an overflow error for some reason. I cannot validate this with random data, so perhaps it has to do with the number of 1's in the values? I've tried to provide some stats which might be helpful...

version 0.51

```python
f0 = numpy.asarray(codon_streams[0])
f1 = numpy.asarray(codon_streams[1])
print type(f0)
print type(f1)
print f0[:20], 'len: %i   min: %i   max: %i' % (len(f0), numpy.min(f0), numpy.max(f0))
print f1[:20], 'len: %i   min: %i   max: %i' % (len(f1), numpy.min(f1), numpy.max(f1))
with sns.axes_style("white"):
    sns.jointplot(f0, f1, kind='hex')
plt.show()
```

Output is:
```
<type 'numpy.ndarray'>
<type 'numpy.ndarray'>
[1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1] len: 4997   min: 1   max: 32
[1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1] len: 4997   min: 1   max: 32
Generating charts...
/usr/local/lib/python2.7/site-packages/matplotlib/tight_layout.py:225: UserWarning: tight_layout : falling back to Agg renderer
  warnings.warn("tight_layout : falling back to Agg renderer")
Traceback (most recent call last):
  File "bitblar5.py", line 435, in <module>
    sns.jointplot(f0, f1, kind='hex')
  File "/usr/local/lib/python2.7/site-packages/seaborn/distributions.py", line 989, in jointplot
    gridsize = int(np.mean([x_bins, y_bins]))
OverflowError: cannot convert float infinity to integer
```

**Comment by mwaskom** [comment]
I believe this was fixed by #403. You should also be able to pass a specific number of histogram bins in `marginal_kws` to avoid the error.

**Comment by conduit242** [comment]
Thanks!
